### ISSUE TITLE
Add aria-labels to LoS buttons

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#6140]
**Current behavior**
The buttons from a LoS do not have a label and are therefore a bit harder to use

**Wanted behavior**
Every button from a LoS should have an aria-label which is the text on the button
